thrift.js

```diff
@@ -413,7 +387,7 @@ Thrift.prototype.resolveIdentifier = function resolveIdentifier(def, identifier,
         return model;
     }
 
-    var parts = identifier.split('.');
+    var parts = name.split('.');
```

Comment:
And this is another name fix, previously "identifier", which would refer to a fat Identifier AST object, whereas this is just the name string from an identifier.